gapic/schema/wrappers.py

```diff
@@ -742,13 +742,14 @@ def field_headers(self) -> Sequence[str]:
     def http_opt(self) -> Dict[str, str]:
         """Return the http option for this method."""
         http = self.options.Extensions[annotations_pb2.http].ListFields()
+        answer: Dict[str, str] = None
```

Comment:
Strictly speaking this breaks the return signature. It would be more consistent to either change the return to be `Optional[Dict[str, str]]` or change the default value of answer to be an empty dictionary.